From 531d4eca2ffb4f9904d8e820dbd2919ac5c76b77 Mon Sep 17 00:00:00 2001
From: Armin Kuster <akuster808@gmail.com>
Date: Mon, 30 Jan 2023 19:33:52 +0000
Subject: [PATCH] odroid-xu3-defconfig: disable CONFIG_BOARD_LATE_INIT

the 2023 now wont boot unless that config option is off.

cros-ec communications failure -96

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Armin Kuster <akuster808@gmail.com>

---
 configs/odroid-xu3_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/odroid-xu3_defconfig b/configs/odroid-xu3_defconfig
index 3f03a9d71f..f4e8e47f78 100644
--- a/configs/odroid-xu3_defconfig
+++ b/configs/odroid-xu3_defconfig
@@ -25,7 +25,6 @@ CONFIG_CONSOLE_MUX=y
 CONFIG_BOARD_TYPES=y
 # CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_DISPLAY_BOARDINFO_LATE=y
-CONFIG_BOARD_LATE_INIT=y
 CONFIG_MISC_INIT_R=y
 CONFIG_SYS_PROMPT="ODROID-XU3 # "
 CONFIG_SYS_PBSIZE=1024
